feat(web-components): added internal silent prop to ic-tooltip

GCHQ-Developer-741 opened the pull request:
## Summary of the changes
Added internal prop to ic-tooltip, called silent. Is used in ic-button to prevent screen readers from reading out the same label twice when both the title and aria-label are applied to an icon button.

Also bumped follow-redirects in multiple packages

## Related issue
#1227 

## Checklist
- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] All acceptance criteria reviewed and met. 
- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.
- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] Browser support tested (Chrome, Safari, Firefox and Edge).
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 
- [x] All prop combinations work without issue. 
- [x] Changes to docs package checked and committed.

ad9242 reviewed (changes requested):
seem to be quite a few changes to other components (badge, toast, back to top). Could we limit it to just the button\tooltip changes?

GCHQ-Developer-741 commented:
> seem to be quite a few changes to other components (badge, toast, back to top). Could we limit it to just the button\tooltip changes?

Functionally, nothing has changed in those components. I had rewritten some code when I was looking into hydration issues involving them and transferred it over when I shelved those tickets. I can remove them if necessary but really there isn't anything to review in them, which is why I added them in
